example/timings/time_forest_partition.cxx

```diff
@@ -382,12 +414,23 @@ main (int argc, char *argv[])
                          "Use a cmesh that tests all tet face-to-face connections."
                          " If this option is used -o is enabled automatically."
                          " Diables -f and -c.");
+  sc_options_add_switch (opt, 'L', "test-linear-cylinder", &test_linear_cylinder,
+                         "Use a linear cmesh to compare linear and occ geometry performance."
+                         " If this option is used -o is enabled automatically."
```

Comment:
If i recall it correctly I copied this description from the `-test-tet` switch, because my tests are located in the same if/else statement. I do not think its true, but then the `-test-tet` description is also not true.

Reply:
You can see that i copied this, because I have the same typo in "Diables" ^^